storage_service/common/datatable_utils.py

```diff
@@ -0,0 +1,159 @@
+"""DataTable utils
```

Comment:
Ah! At some point I thought about using `get_model` to avoid the import from locations, but I wasn't sure about what app label or AppConfig to use and couldn't find much (or any) use of it in the codebase. Maybe it's not a good pattern? I'd appreciate your opinion on that?

Reply:
I think that's a good pattern in some cases (mostly to avoid circular imports) but not necessary here since the datatable implementation is package specific. I haven't seen much of that in the codebase either 🙂

Reply:
Thank you for explaining. Makes sense. I moved the module to the `locations` package.